### PR TITLE
GUVNOR-2270: fix incorrect stack overflow on repository clone

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

File: guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryViewImpl.java [hunk 1]
```diff
@@ -272,7 +272,7 @@ public void setCancelEnabled( final boolean enabled ) {
 
     @Override
     public void setPopupCloseVisible( final boolean closeVisible ) {
-        setPopupCloseVisible( closeVisible );
+        setClosable( false );
```
Comment (manstis) [inline guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryViewImpl.java]: What becomes of the unused parameter ```closeVisible```?

Either pass the parameter to ```setClosable()``` or remove from the method signature.

Comment (porcelli) [inline guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryViewImpl.java]: What are you talking about? :P

**Comment by manstis** [comment]
@porcelli IDK I must have imagined the whole sordid affair.

**Comment by manstis** [comment]
*guvnor*
(master) http://github.com/droolsjbpm/guvnor/commit/afe9d286b

**Pull request closed without merging**